Resources/assets/js/comments.js

```diff
@@ -353,7 +353,7 @@
                 var reply_button_holder = form.closest('.fos_comment_comment_reply');
 
                 var comment_element = form.closest('.fos_comment_comment_show')
-                    .children('.fos_comment_comment_replies');
+                    .find('.fos_comment_comment_replies:first');
```

Comment:
I believe these elements can be nested (replies to replies etc), so that would cause problems in that case. I also think that's why .children() was used in the first place.